packages/entity/src/errors/EntityError.ts

```diff
@@ -0,0 +1,11 @@
+import ES6Error from 'es6-error';
+
+export enum EntityErrorState {
+  UNKNOWN,
+  TRANSIENT,
```

Comment:
In some systems, retryable errors come with information about when to retry (this is most common with 429s). Something to think about.

Reply:
Yeah, not sure we have quite enough information at the entity level to determine what a good retry timeout would be. I think this would probably be better determined at the application layer (since it does the Knex configuration) and then also added to the 429s at that layer.